infra/src/app-service.ts

```diff
@@ -68,7 +68,10 @@ export const service = new kubernetes.core.v1.Service(
         "service.beta.kubernetes.io/aws-load-balancer-ssl-cert": certificateArn,
         "service.beta.kubernetes.io/aws-load-balancer-backend-protocol": "http",
         "service.beta.kubernetes.io/aws-load-balancer-ssl-ports": "443",
-        "service.beta.kubernetes.io/aws-load-balancer-connection-idle-timeout": "60",
+        "service.beta.kubernetes.io/aws-load-balancer-connection-idle-timeout": "900",
+        "service.beta.kubernetes.io/aws-load-balancer-ssl-negotiation-policy":
+          "ELBSecurityPolicy-TLS13-1-2-2021-06",
+        "service.beta.kubernetes.io/aws-load-balancer-type": "nlb",
```

Comment:
Thanks Bohdan. This specific change worries me slightly -- do you know if there's any downtime associated with the change in load balancer?

Reply:
Hmm, provisioning a load balancer takes some time, let me test it on staging to measure the downtime.

Reply:
if EKS is smart it'll bring up the new LB and service and make sure it's live and then cut over all traffic before tearing down the old one. So there may be no downtime at all. But we should test that theory before deploying in prod. :)

Reply:
Just tested it, and it seems there is no downtime.